### ISSUE TITLE
Desktop: Keyboard: Focus Visible

### ISSUE THREAD

**Issue by AutoSponge** [Hashnode/user-feedback#167]
**Describe the bug**
Using a keyboard to tab through interactive elements I'm often unable to determine which elements have focus. Important elements like "Menu" and "Create Account" should have visible focus indicators.

See https://ytho.hashnode.dev/wip-hashnode-ytho-ck7xdp9sq02ejzns1ydc4m3m8#keyboard-review

**To Reproduce**
1. Go to unauthenticated home page.
2. Tab through the interface.
3. Observe lack of focus indication.

**Expected behavior**
I expect clear and consistent focus indication while navigating with a keyboard.

**Screenshots**
See https://ytho.hashnode.dev/wip-hashnode-ytho-ck7xdp9sq02ejzns1ydc4m3m8#keyboard-review

**Desktop (please complete the following information):**
 - OS: macos
 - Browser chrome
 - Version 80

**Additional context**
Link to Focus Visible defn: https://www.w3.org/TR/WCAG22/#focus-visible-enhanced
Example of good focus indication: gov.uk

**Comment by BolajiAyodeji** [comment]
Hi @AutoSponge,

Thanks for the amazing feedback. We are working on getting this fixed and we'll ship an update soon.
Keep the feedback coming :)

**Comment by BolajiAyodeji** [comment]
Hi @AutoSponge, this is fixed now. Thank you for your feedback!
